src/components/orgunits/OrgUnitProfile.js

```diff
@@ -19,24 +19,34 @@ const currentYear = String(new Date().getFullYear())
 const periods = getFixedPeriodsByType(periodType, currentYear)
 const defaultPeriod = filterFuturePeriods(periods)[0] || periods[0]
 
+const ORGUNIT_PROFILE_QUERY = {
+    profile: {
+        resource: 'organisationUnitProfile',
+        id: ({ id }) => `${id}/data`,
+        params: ({ period }) => ({
+            period,
+        }),
```

Comment:
We could remove period here if we do a separate data request in the OrgUnitData component. Maybe we could move all the period handling to this component?

Reply:
I was thinking that too.